easybuild/easyconfigs/v/VMD/VMD-1.9.4a51_configure.patch

```diff
@@ -0,0 +1,167 @@
+--- configure	2021-02-16 12:59:02.511000000 +0100
++++ configure	2021-02-16 12:59:29.660000000 +0100
+@@ -497,17 +497,15 @@
+ 
+ $arch_cc          = "cc";
+ $arch_ccpp        = "CC";
+-$arch_nvcc        = "/usr/local/cuda-10.2/bin/nvcc";
++$arch_nvcc        = "$ENV{'EBROOTCUDA'}/bin/nvcc";
+ $arch_nvccflags   = "-lineinfo --ptxas-options=-v " . 
+-                    "-gencode arch=compute_30,code=compute_30 " .
+-                    "-gencode arch=compute_30,code=sm_35 " .
+-                    "-gencode arch=compute_30,code=sm_37 " .
+                     "-gencode arch=compute_50,code=compute_50 " .
+                     "-gencode arch=compute_50,code=sm_50 " .
+                     "-gencode arch=compute_60,code=compute_60 " .
+                     "-gencode arch=compute_60,code=sm_60 " .
+                     "-gencode arch=compute_70,code=compute_70 " .
+                     "-gencode arch=compute_70,code=sm_70 " .
++                    "-gencode arch=compute_75,code=sm_75 " .
```

Comment:
This should really pick up on the cuda_compute_capabilities template but that can be left out of this PR since it's a bit more work.

Reply:
I completely agree with that. I was thinking the same.

Reply:
We could add version 80 as well since it's supported in CUDA 11 if you want this patch to just add everything just in case.

Reply:
You are right about it. Thanks for reminding me. I had it in mind but later completely forgot it. Here in the center, we have Ampere's. I remember having some problem with one of the packages when I enabled them all. Also, newer CUDA drops support for some older compute_xx.